prov/sm2/src/sm2_ep.c

```diff
@@ -313,7 +312,7 @@ static void sm2_format_cma(struct sm2_xfer_entry *xfer_entry,
 
 static ssize_t sm2_do_cma(struct sm2_ep *ep, struct sm2_region *peer_smr,
 			  sm2_gid_t peer_gid, uint32_t op, uint64_t tag,
-			  uint64_t data, uint64_t op_flags, struct ofi_mr **mr,
```

Comment:
So I know I asked you to do this... but can you actually re-base this change in front of the cma change, and then squash the cma specific part into the cma commit?  This will provide a cleaner development history for a reader.

Reply:
done